src/App.js

```diff
@@ -19,22 +31,166 @@ function App() {
   ]
 
   return (
-    <>
-      <HeaderStyled>Add your activities</HeaderStyled>
-      <Form categories={categories} onSubmit={addActivity} />
-      <TimeRemaining activities={activities} />
-      <Cards activities={activities} categories={categories} />
-    </>
+    <AppGrid>
+      <HeaderStyled>
+        <AppName>Take Your Time</AppName>
+        <Logo src={logo} />
+      </HeaderStyled>
+
+      <ScrollContainer>
+        <TitleStyled>Add your activities</TitleStyled>
+        <OpenModal>
+          <AddStyled onClick={openModal}></AddStyled>
+        </OpenModal>
+
+        <TimeRemaining activities={activities} />
+        <Cards activities={activities} categories={categories} />
+      </ScrollContainer>
+
+      <Footer></Footer>
+
+      <Modal
+        isOpen={modalIsOpen}
+        onRequestClose={closeModal}
+        style={customStyles}
+      >
+        <CloseModal onClick={closeModal}>x</CloseModal>
+        <Form categories={categories} onSubmit={addActivity} />
+      </Modal>
+    </AppGrid>
   )
 
+  function openModal() {
+    setIsOpen(true)
+  }
+
+  function closeModal() {
+    setIsOpen(false)
+  }
+
   function addActivity(activity) {
     setActivities([activity, ...activities])
   }
 }
 
-export default App
+const AppGrid = styled.section`
+  display: grid;
+  grid-template-rows: 48px auto 48px;
+  height: 100vh;
+`
+
+const HeaderStyled = styled.section`
+  background: #1a1919;
+  color: white;
+  height: 48px;
+  width: 100vw;
+  padding: 0 20px;
+  display: flex;
+  align-items: center;
+  font-size: 20px;
+`
+
+const AppName = styled.h1`
+  position: absolute;
+  z-index: 1;
+  font-size: 19px;
+  padding: 12px;
+`
+const Logo = styled.img`
+  position: absolute;
+  width: 42px;
+`
 
-const HeaderStyled = styled.h1`
+const ScrollContainer = styled.main`
+  overflow-y: scroll;
+`
+
+const Footer = styled.section`
+  background: #1a1919;
+  color: white;
+  height: 48px;
+  display: flex;
+  padding-left: 12px;
+  align-items: center;
+  font-size: 20px;
+`
+const TitleStyled = styled.h1`
+  margin: 32px 0 24px 0;
   text-align: center;
   font-size: 21px;
 `
+const OpenModal = styled.button`
+  background: white;
+  border: none;
+  display: block;
+  margin: auto;
+`
+
+const CloseModal = styled.button`
+  border: none;
+  background: #1a1919;
+  color: grey;
+  font-size: 18px;
+  position: absolute;
+  left: 88%;
+  top: 2%;
+
+  :hover {
+    color: white;
+  }
+`
+
+const customStyles = {
+  content: {
+    top: '95%',
+    left: '50%',
+    right: 'auto',
+    bottom: 'auto',
+    marginRight: '-50%',
+    transform: 'translate(-50%, -50%)',
+    border: 'none',
+    background: '#1a1919',
```

Comment:
Hier verstehe ich ehrlich gesagt, nicht ganz was du vorhast. Es wirkt etwas gebastelt. Du könntest es auch kürzer und transparenter lösen. Bspw. indem du Styled Components auch für das Modal verwendest:
```
const ModalStyled = styled(Modal)`
  position: 'absolute';
  border: 'none';
  background: 'rgb(26; 25; 25)';
  border-radius: '4px';
  padding: '20px';
  bottom: 0;
  height: 'max-content';
  width: '100%';
`
```
Und dann eben in der Komponente :
```
<ModalStyled
        isOpen={modalIsOpen}
        onRequestClose={closeModal}
        style={customStyles}
      >
        <CloseModal onClick={closeModal}>x</CloseModal>
        <Form categories={categories} onSubmit={addActivity} />
      </ModalStyled>
```

Das Form-Element könntest du im Styling auch anpassen und auch hier max-content sowie width: 100% verwenden:
```
const FormStyled = styled.form`
  display: flex;
  flex-direction: column;
  align-items: center;
  height: max-content;
  padding-top: 10px;
  width: 100%;
`
```